peeringdb_server/templates/site/register.html

```diff
@@ -78,7 +78,7 @@ <h3>{% trans "Create account" %}</h3>
              data-edit-name="last_name"
              data-edit-type="string" />
 
-      <div class="captcha">
+      <div class=captcha>
```

Comment:
This needs quotes around it (like it was).

Reply:
It's fixed now! :)